docs/tutorials/using_apis/Mbed_USB_WAV_Audio_Player/Mbed_USB_WAV_Audio_Player.md

```diff
@@ -16,39 +20,52 @@ This tutorial uses the USBAudio class of the USB library to send audio data from
-   
+
+1. Copy and paste the example code below into `main.cpp`.
+1. Make sure the SPI pins for the SDBlockDevice object are correct for your board. For example, in the code below, line 10 sets up SPI for the SDBlockDevice for the NXP-K64F Mbed board.
+1. Load a WAV file onto the SD card. The example below uses a public domain WAV file called ["Bach-minuet-in-g.wav"](https://github.com/mrcoulter45/mbed-os-5-docs/raw/Mbed_USB_WAV_Audio_Player_Tutorial/docs/tutorials/using_apis/Mbed_USB_WAV_Audio_Player/Bach-minuet-in-g.wav) that is inside a "songs" directory on the SD card<!--no it isn't; you need to download it and put it on the SD card-->.
+
+    If you want to use another file, it must be WAV, PCM signed 16-bit little endian, or else it will not play (because USBAudio does not support any other WAV formats). The WAV file can have any sampling rate and can have any number of channels. You can use [Online-convert](https://audio.online-convert.com/convert-to-wav) to generate WAV files with different formats from any source audio file. The file "Bach-minuet-in-g.wav" is already in the correct format.
+
    ![Convert audio to WAV](https://raw.githubusercontent.com/ARMmbed/mbed-os-5-docs/development/docs/images/Mbed_USB_WAV_Audio_Player_img1.png)
-   
-1. Compile the program and flash the binary.
-1. Ensuring that the target's USB is connected to the host computer, select `Mbed Audio` as the host PC's default audio input device. On Windows, in Control Panel, set `View by` to `Category` in the top right corner. Then, navigate to `Hardware` and `Sound` > `Sound`, and in the recording tab, right click on `Microphone Mbed Audio` > `Properties` > `Listen`. Check the `Listen to this Device` box, and click `Apply`. The audio from the WAV file is now audible through whichever output device is selected for the host PC's audio output. If `View by` is not set to `Category`, audio will be under Sound in Control Panel:
-   
+
+1. Compile the program and flash the binary.<!--some tutorials, like the alarm tutorial, explain how to do this. Some don't. It might be best to create a page for the tutorials that has building and flashing instructions and link to the troubleshooting section, and then just link all tutorials to it instead of explaining it again and again-->
+1. Make sure that the board is connected to the host PC over USB.
+1. Select `Mbed Audio` as the host PC's default audio input device.
+
+    On Windows, in Control Panel, set `View by` to `Category` in the top right corner. Then, navigate to `Hardware` and `Sound` > `Sound`, and in the recording tab, right click on `Microphone Mbed Audio` > `Properties` > `Listen`. Check the `Listen to this Device` box, and click `Apply`. The audio from the WAV file is now audible through whichever output device is selected for the host PC's audio output. If `View by` is not set to `Category`, audio will be under Sound in Control Panel:
+
```

Comment:
I think we should use **bolding** instead of back ticks for the items shown in the Control Panel UI. At least they are bolded in the Troubleshooting below.

Reply:
Yep. Consistent with how we do things elsewhere.